source/server/microkernel_architecture/microkernel_modules/database_manager/UsersEntity.hpp

```diff
@@ -0,0 +1,39 @@
+#ifndef USERS_ENTITY_HPP_
+# define USERS_ENTITY_HPP_
+
+#include	"Entity.hpp"
+
+//
+// \\\\This class is just an example !///
+// It does not respect the norme
+//
+class UsersEntity : public Entity
+{
+
+  Q_OBJECT
+
+  Q_PROPERTY(int id READ getId WRITE setId)
+  Q_PROPERTY(QString login READ getLogin WRITE setLogin)
+  Q_PROPERTY(QString password READ getPassword WRITE setPassword)
+
+public:
+  UsersEntity();
+  virtual ~UsersEntity();
+
+protected:
+  int		id;
+  QString	login;
+  QString	password;
```

Comment:
No it's not, these variables have to keep the exact same name as the column name in the database.